### ISSUE TITLE
Flatpak 1.0 marketing readiness

### ISSUE THREAD

**Issue by matthiasclasen** [flatpak/flatpak.github.io#252]
We should have a checklist of things to have in place for a 1.0 release:

Create materials;

 - [x] [Release notes drafted](https://github.com/flatpak/flatpak/wiki/1.0-Release-Notes)
 - [x] 1.0 press release written
 - [x] [New publicity video completed](https://github.com/flatpak/flatpak/issues/1892)

Prepare hosting of new materials

 - [x] [New press contact email address](https://bugzilla.gnome.org/show_bug.cgi?id=796956)
 - [x] New press page - #288 
 - [x] Add new press release to the website
 - [x] Add the video to the website, probably on the home page - #290

Other website issues it would be nice to have fixed:

 - [ ] Update the quotes on the home page
 - [x] Improve the about page - #254 
 - [x] Add CentOS - #270
 - [ ] Add list of presentations - #232
 - [x] Some home page improvements from #289

**Comment by matthiasclasen** [comment]
For the press release, see https://github.com/flatpak/flatpak/wiki/1.0-Press-Release

**Comment by allanday** [comment]
I'm still waiting on that wiki page to be populated - I need some technical details of what will be in the 1.0 release in order to write about it.

**Comment by allanday** [comment]
Updated the to do list, to (hopefully) reflect where we're at today. 1.0 is close!

**Comment by allanday** [comment]
Might be nice to give the home page a refresh too; I know that @jimmac wasn't particularly happy with it. There probably isn't time for that though.

**Comment by allanday** [comment]
@alexlarsson has updated the release notes from his perspective, and I've given them another review for style and language. If anyone wants to proof read them, that would be helpful.

**Comment by alexlarsson** [comment]
I've got the actual code basically frozen now. Running various tests on the final results

**Comment by allanday** [comment]
Some small issues that we need to sort out with the flatpak-1-0 branch before it goes live:

 - [ ] The blue colour of "Apps on Linux" looks a bit odd to me; it's unclear why that text has a different colour
 - [ ] There's no explanation of what the app icons are or why they're there. The existing live site says "choose from a growing collection of apps" above them.
 - [ ] The press page needs a download link to logo assets, as well as information on the licence.
 - [ ] I added a history section to the about page, but it needs checking over and refining. Also, the layout isn't great. If we don't have this ready in time, we can easily cut it.

**Comment by allanday** [comment]
It's alive!
